tutor/templates/local/docker-compose.yml

```diff
@@ -150,7 +150,7 @@ services:
       SERVICE_VARIANT: lms
       SETTINGS: ${EDX_PLATFORM_SETTINGS:-tutor.production}
       C_FORCE_ROOT: "1" # run celery tasks as root #nofear
-    command: ./manage.py lms celery worker --loglevel=info --hostname=edx.lms.core.default.%%h --maxtasksperchild 100
+    command: ./manage.py lms celery worker --logfile=/openedx/logs/edx.log --loglevel=info --hostname=edx.lms.core.default.%%h --maxtasksperchild 100
```

Comment:
With this proposed change, celery will still log to stdout by default, because edx.log is symlinked to /dev/stdout.
Do you really need to collect logs from celery? The logs impacted by the `--logfile` option are only the logs emitted by the `celery worker` command, not the regular application logs. On my machine, these logs are not stored in `all.log`:

```
  -------------- celery@edx.lms.core.default.%491fdd2d8d19 v3.1.25 (Cipater)                                                                                                                   
---- **** ----- 
--- * ***  * -- Linux-4.15.0-48-generic-x86_64-with-Ubuntu-16.04-xenial
-- * - **** --- 
- ** ---------- [config]
- ** ---------- .> app:         proj:0x7f398f189850
- ** ---------- .> transport:   amqp://guest:**@rabbitmq:5672//
- ** ---------- .> results:     
- *** --- * --- .> concurrency: 4 (prefork)
-- ******* ---- 
--- ***** ----- [queues]
 -------------- .> edx.lms.core.default exchange=edx.lms.core(direct) key=edx.lms.core.default
                .> edx.lms.core.high exchange=edx.lms.core(direct) key=edx.lms.core.high
                .> edx.lms.core.high_mem exchange=edx.lms.core(direct) key=edx.lms.core.high_mem

[tasks]
  . bulk_email.tasks.send_course_email
  . celery_utils.tasks.mark_resolved
  . edx_sga.tasks.zip_student_submissions
  . email_marketing.tasks.get_email_cookies_via_sailthru
  . email_marketing.tasks.update_course_enrollment
...
```

I think we can safely keep these logs in stdout, right?